src/ui.button/index.vue

```diff
@@ -157,6 +162,38 @@ const props = defineProps({
     type: String,
     default: "",
   },
+
+  /**
+   * Icon to be displayed on the left side.
+   */
+  iconLeft: {
+    type: String,
+    default: "",
+  },
+
+  /**
+   * Icon to be displayed on the right side.
+   */
+  iconRight: {
+    type: String,
+    default: "",
+  },
+
+  /**
+   * Left icon color.
+   */
+  iconLeftColor: {
```

Comment:
color should be applied same as in button, for primary variant should be white. 

The props no need.